### PR TITLE
ddl: fix ddl test fail by paralle test

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Fix ddl test fail by paralle test.

### What is changed and how it works?
add check.

### Check List 

Tests 
 - No code

Code changes

Side effects

Related changes

### CODE REVIEW

File: ddl/ddl_worker_test.go [hunk 1]
```diff
@@ -450,6 +450,9 @@ func (s *testDDLSuite) TestCancelJob(c *C) {
 		if job.State == model.JobStateSynced || job.State == model.JobStateCancelled || job.State == model.JobStateCancelling {
 			return
 		}
+		if len(test.jobIDs) > 0 && test.jobIDs[0] != job.ID {
```
Comment (zimulala) [inline ddl/ddl_worker_test.go]: Why add this check？Add some comments？

Comment (crazycs520) [reply ddl/ddl_worker_test.go]: done.

**Pull request merged**